# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hotplug++/silence.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- hotplug++-0.1.1/bin/main.cc.vanilla	2007-01-16 11:23:51.213501013 +0100
+++ hotplug++-0.1.1/bin/main.cc	2007-01-16 11:24:05.334305712 +0100
@@ -45,9 +45,12 @@
   
   if (mode.empty() && subsystem.empty())
     mode = "list";
-  
+
+#if 0  
   std::cout << "hotplug++: mode: '" << mode
 	    << "', subsystem: '" << subsystem << "'" << std::endl;
+
+#endif
   
   std::vector<basic_subsystem*> subsystems;
   // fill the subsystems vector
